docs/operator.md

```diff
@@ -332,14 +332,79 @@ What to do with a Pod if it doesn't satisfy the [Kubernetes Pod Topology Spread
 | ----------- | ---------- |
 | :material-code-string: string     | `DoNotSchedule` |
 
-### `mysql.expose.enabled`
+### `mysql.exposePrimary.enabled`
 
-Enable or disable exposing Percona Server for MySQL nodes with dedicated IP addresses.
+Enable or disable exposing Percona Server for MySQL nodes with dedicated IP addresses for cluster type group-replication.
```

Comment:
@gkech i think this field is also used with async replication, right?

Reply:
We can write this line like this:

> Enable or disable exposing the primary Percona Server for MySQL node with dedicated IP addresses.